deployment/ofed-driver-pod.yaml

```diff
@@ -18,10 +18,25 @@ spec:
           mountPropagation: Bidirectional
         - name: etc-network
           mountPath: /etc/network
+        - name: host-etc
```

Comment:
perhaps prefix host paths with `/host` ?  for clarity

to me it seems abit unclear to find `etc` folder under `opt`